pydatastructs/trees/tests/test_binary_trees.py

```diff
@@ -54,6 +54,13 @@ def test_BinarySearchTree():
     assert b.delete(-10) is True
     assert b.delete(-3) is True
     assert b.delete(-13) is None
+    bl = BST()
+    nodes = [50, 30, 90, 70, 100, 60, 80, 55]
+    for node in nodes:
+        bl.insert(node, node)
+    assert bl.tree[bl.lowest_common_ancestor(80, 55, 2)].key == 70
+    assert bl.tree[bl.lowest_common_ancestor(60, 70, 2)].key == 70
+    assert bl.lowest_common_ancestor(-3, 4) is None
```

Comment:
Please add more tests. The diff coverage is low. The lines in red in [this report](https://codecov.io/gh/codezonediitj/pydatastructs/pull/88/diff) aren't executed at least once. Add tests to ensure they are executed at least once.
You can install `pytest-cov` locally. See, `.travis.yml` for commands to be used for getting the coverage report along with the tests run.

Reply:
Yes. I'll start working on that. Thanks.